src/wiktextract/extractor/en/page.py

```diff
@@ -3538,7 +3539,9 @@ def usex_template_fn(name, ht):
                             lines = [parts[0].strip()]
                             tr = parts[1].strip()
                 elif len(lines) > 1:
-                    if any(re.search(r"[]\d:)]\s*$", x) for x in lines[:-1]):
+                    if any(
+                        re.search(r"[]\d:)]\s*$", x) for x in lines[:-1]
+                    ) and example_template_names in (["zh-x"], ["zh-usex"]):
```

Comment:
Should be `not in`.

This might be difficult, but I think it's better to put the "zh-x" code in a new function so it could be reused for  etymology section. And process the expanded HTML tags would fix the unreliable `classify_desc` problem.

Reply:
Thanks for spotting that, I was hasty.

The classify_desc problem isn't with HTML tags, it's because classify_desc breaks in square brackets, like "[Chinese language name, trad.]".

Reply:
I mean if we could processing HTML tags then separating the text would be easier. Because language variety and other texts can be distinguished through HTML tag name and CSS class names. This is just a suggestion, if we want to move the data inside brackets to tags in the future, we could copy the zh edition code, they use the same HTML tags.